themes/bootstrap3/js/advanced_search.js

```diff
@@ -137,6 +137,7 @@ deleteGroup = function _deleteGroup(group) {
 $(document).ready(function advSearchReady() {
   $('.clear-btn').click(function clearBtnClick() {
     $('input[type="text"]').val('');
+    $('input[type="checkbox"]').prop("checked", false);
```

Comment:
EDS can have checkboxes that are checked by default. I don't think it's appropriate to clear them. All checkboxes should rather be reset to the initial state.

Reply:
Yes, I believe that's the clearest way to do it.

Reply:
Thanks for adding this, @xrosecky -- so I think now the next step is adding the "reset-checked" attribute where it is needed. I'll look into this next week when I have a bit more time, unless somebody else gets there first. :-)

Reply:
It is OK for me.

Reply:
Thanks, @xrosecky. Do you have any opinions about the discussion below about renaming `reset-checked` to `initially-checked` or `checked-by-default` for clarity?

Reply:
`checked-by-default ` seems better to me.

Reply:
Thanks, @xrosecky, I have made the change and will merge this as soon as CI passes.